lib/src/svg/colors.dart

```diff
@@ -50,19 +50,77 @@ Color parseColor(String colorString) {
     return Color.fromRGBO(rgb[0], rgb[1], rgb[2], opacity);
   }
 
+  // Conversion code from: https://github.com/MichaelFenwick/Color, thanks :)
+  if (colorString.toLowerCase().startsWith('hsl')) {
+    List<num> rgb = <num>[0, 0 , 0];
```

Comment:
Please use either `int` or `double` here (avoid `num`) and elsewhere, as appropriate.

Also, pleae `dartfmt` this code.

Reply:
Also, please move the declaration of this down closer to where it is used.